tests/functional/test_cluster_functions.py

```diff
@@ -306,43 +328,46 @@ def test_add_unknown_host_to_cluster(self, client):
             err.HOST_NOT_FOUND.equal(e, 'host doesn\'t exist')
         steps.delete_all_data(client)
 
-    def test_add_host_in_two_diff_clusters(self, client):
-        prototype = utils.get_random_cluster_prototype(client)
+    def test_add_host_in_two_diff_clusters(self, sdk_client_fs: ADCMClient):
+        bundle = sdk_client_fs.upload_from_fs(get_data_dir(__file__, 'cluster_simple'))
+        bundle_hp = sdk_client_fs.upload_from_fs(get_data_dir(__file__, 'hostprovider_bundle'))
         with allure.step('Create cluster1'):
```

Comment:
There is no need for that, bundle.cluster_create already do this message.

Actually there are a lot of places in adcm_client where we have allure decorators. Check out adcm_client code to find them.